tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -63,6 +67,12 @@ def _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, os):
     )
 
 
+def get_fsx_fs_id(cluster, region):
+    cluster_name = "parallelcluster-{0}".format(cluster.name)
```

Comment:
`cluster.cfn_name` would give you the same thing

Reply:
changed